random-reviewer-picker/main.js

```diff
@@ -73,6 +74,10 @@ async function pickUsers(userList, pickAmount, removeBusy) {
     filteredList = userList;
   }
 
+  if (filteredList.indexOf(prOwner) >= 0) {
+    filteredList.splice(filteredList.indexOf(prOwner));
```

Comment:
⚠️ There's a `1` missing as a second parameter here

Reply:
Thank you